docs/nodes/list_main/match.rst

```diff
@@ -34,7 +34,7 @@ Parameters
 * **Short**: cut all to minimal length of lists
 * **Cycle**: extend all to maximal length of lists by cyclingly repeating
 * **Repeat**: extend all to maximal length of lists by last item repeating
-* **X-Ref**: extend all to maximal length of lists by multiply lengthes,
+* **X-Ref**: extend all to maximal length of lists by multiply lengths,
```

Comment:
Perhaps "...by muliply**ing** lengths" was meant here?

Reply:
It seems so

Reply:
Fixed in [2c0c522](https://github.com/nortikin/sverchok/pull/4900/commits/2c0c5224fa151524a4007ce2596a0c3afafc9001)